README.md

```diff
@@ -14,13 +14,19 @@ TBA
 
 ### Running with Docker
 
-1. git clone --recursive https://github.com/Sage-Bionetworks/rocc-app.git
+1. Clone this repository, as well as the two linked repositories.  This is done
+with the `--recursive` flag.
+
+       git clone --recursive https://github.com/Sage-Bionetworks/rocc-app.git
+
+Refer to **Updating a Linked Repository** section below if you need to update one
```

Comment:
Use an anchor that allows the user to jump to this section.

Reply:
For a future update

Reply:
Thanks for the suggestion.

Done :white_check_mark: